jdaviz/configs/imviz/plugins/tools.py

```diff
@@ -183,6 +185,10 @@ def _dragevent(self, data):
             self._layer_state.contrast = contrast
 
     def on_mouse_or_key_event(self, data):
+        if (time.time() - self._time_last) <= 0.15:
```

Comment:
As discussed during tag-up, this re-adds something that was removed in #1403 . Duy mentioned they are going to look again but on the sliders code instead.